docs/swagger.yaml

```
@@ -886,6 +923,162 @@ paths:
+      summary: Assign a tag to an article
+      description: Assign a tag to an article
+      parameters:
+      - in : path
+        name : articleId
+        schema:
+          type: integer
+        required: true
+        description : Numeric ID of the article to get
+      requestBody:
+        description: Assign a tag to an article
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                tagId: 
+                  type: integer
+                  description: tag id
+                  example: 10
+      responses:
+        '201':
+          description: Successfully tagging an article
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    description: successfully tagging an article
+                    example: success
+                  data:
+                    type: object
+                    description: Data for tagging an article
+                    properties:
+                      message:
+                        type: string
+                        description: Message for tagging an article
+                        example: Article  successfully tagged
+                      tagId: 
+                        type: integer
+                        description: tag id
+                        example: 10
+                      articleId: 
+                        type: integer
+                        description: article id
+                        example: 10
+  /articles/{articleId}/tags/{tagId}:
+    delete:
+      tags:
+        - article
+      operationId: Delete a tag assigned to an article
+      summary: delete a tag assigned to a tag to an article
+      description: delete a tag assigned to a tag to an article
+      parameters:
+      - in : path
+        name : articleId
+        schema:
+          type: integer
+        required: true
+      - in : path
+        name : tagId
+        schema:
+          type: integer
+        required: true
+        description : Numeric ID of the article and tag
+      responses:
+        '200':
+          description: Successfully untagging an article
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    description: successfully untagging an article
+                    example: success
+                  data:
+                    type: object
+                    description: Data for untagging an article
+                    properties:
+                      message:
+                        type: string
+                        description: Message for untagging an article
+                        example: Tag has been removed from post
+  /articles/query?tag={tagId}:
```

query params are not defined like path parameters, it would just be the normal url

```suggestion
  /articles/query
```

---

wouldn't it be difficult for a beginner to use the endpoint if i just used `/articles/query `instead of `/articles/query?tag=tagId` in the documentation?